saleor/graphql/order/filters.py

```diff
@@ -279,19 +283,24 @@ def filter_has_fulfillments(qs, value):
 
 
 def filter_fulfillments(qs, value):
-    if value is None:
+    if not value:
         return qs.none()
-    fulfillment_qs = None
-    if status_value := value.get("status"):
-        fulfillment_qs = filter_where_by_value_field(
-            Fulfillment.objects.using(qs.db), "status", status_value
-        )
-    if metadata_value := value.get("metadata"):
-        fulfillment_qs = filter_where_metadata(
-            fulfillment_qs or Fulfillment.objects.using(qs.db), None, metadata_value
-        )
-    if fulfillment_qs is not None:
-        return qs.filter(Exists(fulfillment_qs.filter(order_id=OuterRef("id"))))
+
+    lookup = Q()
+    for input_data in value:
+        fulfillment_qs = None
```

Comment:
Does it mean, that we will allow to pass empty object? like: value: [ {}, {"status": "..."}] 🤔 
If yes then I think this should be filtered out and we should raise exception (IMHO it could be a. separate task). 
But for now I think we could have something similar to `filter_where_evetns`: 
```python
    for input_data in value:
        if not {"date", "type"}.intersection(input_data.keys()):
            return qs.none()
```

Reply:
I have conditions before each filter  like: `if status_value := input_data.get("status"):`
so it won't be this way. I don't think that we should raise en error in such case, IMO returning empty list is ok